ui/plugins/controller.js

```diff
@@ -76,11 +77,15 @@ treeherder.controller('PluginCtrl', [
                     job_id,
                     {timeout: selectJobPromise});
 
+                var jobIdPromise = PhSeries.getSeriesByJobId(
+                    $scope.repoName, job_id);
+
```

Comment:
job id been used in here, and you can see the job id has been already been defined in the codebase which I only need to use it. @jmaher